docs/guides/react-server-cli.md

```diff
@@ -132,58 +219,72 @@ Minify client JavaScript and CSS.
 Defaults to **false** in development mode and **true** in production.
 
 #### --long-term-caching
-Adds hashes to all JavaScript and CSS file names output by the build, allowing for the static files to be served with far-future expires headers. This option is incompatible with --hot.
+Adds hashes to all JavaScript and CSS file names output by the build, allowing
+for the static files to be served with far-future expires headers. This option
+is incompatible with --hot.
 
 Defaults to **false** in development mode and **true** in production.
 
-#### --compile-only
-Compile the client JavaScript only, and don't start any servers. This is what you want to do if you are building the client JavaScript to be hosted on a CDN or separate server. Unless you have a very specific reason, it's almost always a good idea to only do this in production mode.
-
-Defaults to **false**.
-
 #### --js-url
-A URL base for the pre-compiled client JavaScript; usually this is a base URL on a CDN or separate server. Setting a value for js-url means that react-server-cli will not compile the client JavaScript at all, and it will not serve up any of the client JavaScript. Obviously, this means that --js-url overrides and ignores all of the options related to JavaScript compilation and serving: --hot, --js-port, and --minify.
+A URL base for the pre-compiled client JavaScript; usually this is a base URL
+on a CDN or separate server. Setting a value for js-url means that
+react-server-cli will not compile the client JavaScript at all, and it will
+not serve up any of the client JavaScript. Obviously, this means that --js-url
+overrides and ignores all of the options related to JavaScript compilation and
+serving: --hot, --js-port, and --minify.
 
 Defaults to **null**.
 
 #### --https
 
-If true, the server will start up using https with a self-signed certificate. Note that browsers do not trust self-signed certificates by default, so you will have to click through some warning screens. This is a quick and dirty way to test HTTPS, but it has some limitations and should never be used in production. Requires OpenSSL to be installed.
+If true, the server will start up using https with a self-signed certificate.
+Note that browsers do not trust self-signed certificates by default, so you
+will have to click through some warning screens. This is a quick and dirty way
+to test HTTPS, but it has some limitations and should never be used in
+production. Requires OpenSSL to be installed.
```

Comment:
Oh... I did a bunch of long-line shortening here, so I could see what I was editing.  I should have done that in a dedicated commit.